pkg/devfile/adapters/common/utils.go

```diff
@@ -32,7 +32,7 @@ const (
 
 	// Default Image that will be used containing the supervisord binary and assembly scripts
 	// use GetBootstrapperImage() function instead of this variable
-	defaultBootstrapperImage = "registry.access.redhat.com/openshiftdo/odo-init-image-rhel7:1.1.3"
+	defaultBootstrapperImage = "registry.redhat.io/ocp-tools-4/odo-init-container-rhel8:1.1.5"
```

Comment:
This doesn't look like. 
`registry.redhat.io` is a registry that requires authentication.

Reply:
Woops!

Reply:
Can you update registry to registry.access.redhat.com